javalib/src/main/scala/java/util/LinkedHashMap.scala

```diff
@@ -0,0 +1,60 @@
+package java.util
+
+import java.{util => ju}
+
+import scala.collection.{mutable => mu}
+
+
+class LinkedHashMap[K, V] private (inner: mu.LinkedHashMap[Box[K], V],
+      initialCapacity: Int, loadFactor: Float, accessOrder: Boolean)
+      extends HashMap[K, V](inner, initialCapacity, loadFactor) {
+
+  self =>
+
+  def this(initialCapacity: Int, loadFactor: Float, accessOrder: Boolean) =
+    this(mu.LinkedHashMap.empty[Box[K], V], initialCapacity, loadFactor, accessOrder)
+
+  def this(initialCapacity: Int, loadFactor: Float) =
+    this(initialCapacity, loadFactor, false)
+
+  def this(initialCapacity: Int) =
+    this(initialCapacity, LinkedHashMap.DEFAULT_LOAD_FACTOR)
+
+  def this() =
+    this(LinkedHashMap.DEFAULT_INITIAL_CAPACITY)
+
+  def this(m: Map[_ <: K, _ <: V]) = {
+    this()
+    initFrom(m)
+  }
+
+  override def get(key: scala.Any): V = {
+    val value = super.get(key)
+    if (accessOrder) {
+      val boxedKey = Box(key.asInstanceOf[K])
+      if (value != null || containsKey(boxedKey)) {
+        inner.remove(boxedKey)
+        inner(boxedKey) = value
```

Comment:
Yes, it will. 

The doc. says: "A special constructor is provided to create a linked hash map whose order of iteration is the order in which its entries were last accessed, from least-recently accessed to most-recently (access-order). This kind of map is well-suited to building LRU caches."